### PR TITLE
AM-3004 : Add hearing-viewer & hearing-manager to CTSC users for SSCS

### PR DESCRIPTION
**Pull request by prashanthkumarnb**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/AM-3004

### Change description ###
This PR is Dependent  on RAS hmcts/am-role-assignment-service#2091

> NB: This PR has temporary changes to Jenkins and Chart file configuration that need reversing before merger.

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -75,7 +75,8 @@ env.PACT_BROKER_FULL_URL = "https://pact-broker.platform.hmcts.net"
 env.AMQP_HOST = "rd-servicebus-aat.servicebus.windows.net"
 env.CRD_TOPIC_NAME = "rd-caseworker-topic-aat"
 env.JRD_TOPIC_NAME = "rd-judicial-topic-aat"
-env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
+env.ROLE_ASSIGNMENT_URL = "https://am-role-assignment-service-pr-2091.preview.platform.hmcts.net"
+//AM-3004 env.ROLE_ASSIGNMENT_URL = "http://am-role-assignment-service-aat.service.core-compute-aat.internal"
```
Comment (mattnayler) [inline Jenkinsfile_CNP:79]: NB: This temporary change must be reverted before merging.

File: charts/am-org-role-mapping-service/values.preview.template.yaml [hunk 2]
```diff
@@ -34,7 +34,8 @@ java:
   environment:
     IDAM_S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
-    ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    # AM-3004 ROLE_ASSIGNMENT_APP_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
+    ROLE_ASSIGNMENT_APP_URL: https://am-role-assignment-service-pr-2091.preview.platform.hmcts.net
```
Comment (mattnayler) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:38]: NB: This temporary change must be reverted before merging.

**Review by mattnayler — DISMISSED** [review]
LGTM

**Review by mikebrownccd — DISMISSED** [review]
Still needs to have the temp url reverted before merging into Master

File: Jenkinsfile_CNP (hunk 1 above)
Comment (mikebrownccd) [reply Jenkinsfile_CNP]: Reverted

File: charts/am-org-role-mapping-service/values.preview.template.yaml (hunk 2 above)
Comment (mikebrownccd) [reply charts/am-org-role-mapping-service/values.preview.template.yaml]: Reverted

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Review by mattnayler — APPROVED** [review]
Re-approved after preview config change 2dfd7f76a63f0d87f2fa6d1b01bc9f5b71f27ea1.

**Pull request merged**